Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -292,6 +292,53 @@ function MiG_29A:defineBrakeLever(identifier, iCommand, arg_number, category, de
 	return control
 end
 
+-- Unit system set by the user settings, "metric" or "imperial"
+local unitSystem
+
+MiG_29A:addExportHook(function(dev0)
+	local val = dev0:get_argument_value(1)
+
+	if unitSystem == nil then
```

Comment:
I guess this assumes the unit system can't be changed mid-game. If that is true, then you could probably move val inside the unitsystem if statement

Reply:
Indeed it must be changed in the main menu of DCS, I moved the val it should be in the if as you said.